app/views/manuals/show.html.erb

```diff
@@ -117,9 +117,11 @@
     <% end %>
 
     <% if manual.published? %>
```

Comment:
@ryanb-gds We discussed this at some point, I agree that there is no point to show the link if there is nothing published at all.